api/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'java-library'
 apply plugin: "com.vanniktech.maven.publish"
 
 dependencies {
-    implementation 'com.fasterxml.jackson.core:jackson-annotations:2.9.0'
+    api 'com.fasterxml.jackson.core:jackson-annotations:2.9.0'
```

Comment:
Why does this have to change?

Reply:
It is also being used by the 'service' module.